src/fsverity/ioctl.rs

```diff
@@ -1,10 +1,22 @@
 use std::os::fd::AsFd;
 
-use anyhow::Result;
+use rustix::io::Errno;
 use rustix::ioctl;
+use thiserror::Error;
 
 use super::FsVerityHashValue;
 
+/// Measuring fsverity failed.
+#[derive(Error, Debug)]
+pub enum MeasureVerityError {
+    #[error("i/o error")]
+    Io(#[from] std::io::Error),
+    #[error("Expected algorithm {expected}, found {found}")]
+    InvalidDigestAlgorithm { expected: u16, found: u16 },
+    #[error("Expected digest size {expected}")]
+    InvalidDigestSize { expected: u16 },
```

Comment:
I feel like this could be mixed with `InvalidDigestAlgorithm`.  I'm not sure how useful it is to have those integers included there.

Reply:
Yeah the chance of anything external using them is probably nil. But if one is going to use thiserror, it's pretty cheap to add this type of stuff and in particular it becomes useful for tests to precisely validate things.

I don't feel strongly though, if you want to drop them we can